types/fastly:cache.d.ts

```diff
@@ -1,11 +1,16 @@
 declare module "fastly:cache" {
+
+  interface PurgeOptions {
+    scope: "pop" | "global"
```

Comment:
It would be great to start documenting types. Ideally we'd generate the mdx from the types even.

Reply:
We originally did generate docs from types but it didn't give us as much expressive documentation capabilities as were we wanting to have

Reply:
Nothing can beat the user experience of having docs in the types though as well.